benchmark/src/main/content/bin/kc-rolling-restart.sh

```diff
@@ -1,34 +1,47 @@
 #!/usr/bin/env bash
-# Use this for deleting all Keycloak pods in sequence.
+# Use this for deleting all Keycloak or Infinispan pods in sequence.
 set -e
 
 if [[ "$RUNNER_DEBUG" == "1" ]]; then
   set -x
 fi
 
 : ${PROJECT:="runner-keycloak"}
+: ${POD_LABEL:="keycloak"}  # Default to keycloak, can be changed via command line
+
+# Check for input arguments to determine pod label (keycloak or infinispan-pod)
+if [[ "$1" == "infinispan" ]]; then
+  POD_LABEL="infinispan-pod"
+elif [[ "$1" != "" && "$1" != "keycloak" ]]; then
+  echo "Invalid argument. Please specify either 'keycloak' or 'infinispan'."
+  exit 1
+fi
```

Comment:
my 2 cents, I would remove this and rely only on `POD_LABEL` to be set before invoking the script.

Reply:
fair enough, I will update the script to do that.